addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -181,8 +181,8 @@ def _generate_efaktur_invoice(self, delimiter):
 
             lines = move.line_ids.filtered(lambda x: x.product_id.id == int(dp_product_id) and x.price_unit < 0 and not x.display_type)
             eTax['FG_UANG_MUKA'] = 0
-            eTax['UANG_MUKA_DPP'] = int(abs(sum(lines.mapped('price_subtotal'))))
-            eTax['UANG_MUKA_PPN'] = int(abs(sum(lines.mapped(lambda l: l.price_total - l.price_subtotal))))
+            eTax['UANG_MUKA_DPP'] = int(round(abs(sum(lines.mapped('price_subtotal'))), 0))
+            eTax['UANG_MUKA_PPN'] = int(round(abs(sum(lines.mapped(lambda l: l.price_total - l.price_subtotal))), 0))
```

Comment:
Shouldn't all the values be rounded before doing the sum?

Reply:
Good point, we will then have the same behavior as for the invoice :+1: